jwt.js

```diff
@@ -8,6 +8,7 @@ const {
   BadRequest,
   Unauthorized
 } = require('http-errors')
+let secretKey, secretPass
```

Comment:
those should not be defined at module level, but they should be wrapped in the plugin

Reply:
^ this still needs a fix.